### ISSUE TITLE
Some technical questions

### ISSUE THREAD

**Issue by gabriel-wainmann** [chenguolin/MoVieS#3]
This is wonderful work. Thank you for sharing.

1. Is the depth analysis temporally consistent?
2. Does the system produce metric distances and not only relative distances?

Thanks!

**Comment by chenguolin** [comment]
Hi @gabriel-wainmann, Thanks for your interest!

1. Yes. The depth head is trained on video depth values, so they are expected to be consistent temporally.
2. No. As the pretrained backbone VGGT processes normalized point clouds, all samples are normalized to have a unit mean norm. However, **all datasets used in this work are in metric**, and we also tried to fine-tune VGGT and train MoVieS in the metric scale, but this led to a decline in reconstruction quality. So we followed the normalized setting in VGGT.

**Comment by gabriel-wainmann** [comment]
Thank you.


Gabriel Wainmann [image: LinkedIn Professional Profile]
<http://www.linkedin.com/in/gabrielwainmann/>
